ui/src/pages/backoffice/Document/DocumentDetails/DocumentHeader.js

```diff
@@ -29,15 +29,7 @@ export class DocumentHeader extends Component {
         <br />
         <label className="muted">Created on</label> {toShortDate(data.created)}
         <br />
-        {!data.metadata.open_access && (
-          <>
-            <Label size="large" color="red">
-              <Icon name="lock" />
-              Restricted access
-            </Label>
-            <br />
-          </>
-        )}
+        <RestrictedLabel openAccess={data.metadata.open_access} />
```

Comment:
This is way I was trying to convince the library to stick to one convention of restricted. Instead, we will have `open_access` for e-items, but `restricted` for Documents with opposite boolean values.
I would suggest to maybe rename this component `OpenAccess` then, because we will have the `Restricted` one for the document probably. WDYT?

Reply:
I solved it in the next PR as it was a bit too much for one